examples/token-erc20/package.json

```diff
@@ -4,13 +4,16 @@
   "scripts": {
     "dev": "ponder dev",
     "start": "ponder start",
-    "codegen": "ponder codegen"
+    "codegen": "ponder codegen",
+    "lint": "eslint . --debug"
```

Comment:
Can remove the `--debug`. Also, all of the examples should have this lint script as I mentioned below.